commit 99c31b5ecc12968f36f72790050e230e0619f2f8 (refs/changes/00/400/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-02T10:55:33-07:00 (3 years, 1 month ago)
    
    MANTA-2863 manta zone setup scripts need to wait for config-agent to start
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/services.sh b/services.sh
index 5adc58c..a04dd6f 100644
--- a/services.sh
+++ b/services.sh
@@ -117,7 +117,7 @@ function manta_download_metadata {
 #
 function manta_enable_config_agent {
     svccfg import /opt/smartdc/config-agent/smf/manifests/config-agent.xml
-    svcadm enable config-agent
+    svcadm enable -s config-agent
 }
 
 
